src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -741,27 +741,12 @@ else if("border".equalsIgnoreCase(element))
 		else if("border-radius".equalsIgnoreCase(element))
 		{
 			auxilaryVerifiers[65]=new CSSPropertyVerifier(Arrays.asList("/"),null,null,null,true);
-			elementVerifiers.put(element,new CSSPropertyVerifier(null,ElementInfo.VISUALMEDIA,null,Arrays.asList("64<1,4>", "64<1,4> 65 64<1,4>")));
+			elementVerifiers.put(element,new CSSPropertyVerifier(null,ElementInfo.VISUALMEDIA,null,Arrays.asList("40<1,4>", "40<1,4> 65 40<1,4>")));
```

Comment:
[40] is only added conditionally for padding (see line 1353), 64 always exists.

Reply:
[40] always exists since c5ddfc86935432cc01e0daf35bba13e4fd752a3c

Reply:
Ok — I missed that. Thanks!